specification/overview.adoc

```diff
@@ -28,7 +28,8 @@ managed by the hosting supervisor domain (OS/VMM) include memory, CPU, I/O
 resources and platform capabilities required to host the TVM workload. The terms
 hosting supervisor domain and OS/VMM are used interchangeably in this
 specification. The underlying memory isolation mechanisms for supervisor domains
-(Smmtt) is agnostic of the number of supervisor domains. On processors that 
+(Smmtt) is agnostic of the number of supervisor domains. On processors with 
+hypervisor mode that 
```

Comment:
nit: hypervisor(h) extension ?

Reply:
Thanks for reviewing. This text was already rewritten in one of the previous commits. Could you please have a look to the newest version?

Reply:
My bad. I see it now. I am not sure why github was showing me the older commits.